frontend/src/components/dialog/DialogActivityEdit.vue

```diff
@@ -56,6 +57,7 @@ export default {
     }
   },
   methods: {
+    /*
```

Comment:
Auch hier, wieder aktivieren oder ganz löschen

Reply:
To be fixed in #2350